contrib/fwupd.spec.in

```diff
@@ -421,130 +397,13 @@ done
 %endif
 %{_libdir}/libfwupd.so.2*
 %{_libdir}/libfwupdplugin.so.%{fwupdplugin_version}*
+%{_libdir}/libfwupdpluginbuiltin.so.%{fwupdplugin_version}*
```

Comment:
Could we keep this somewhere that other stuff doesn't have a chance to link with it like we did with plugins?

Reply:
Even if it could link with it, the daemon wouldn't instantiate the GType as it's not going to be in the static array in `fu-plugin-builtin.h` -- _evil laugh_.

Reply:
> So since it's a "private" library it shouldn't be stored there.

Right, agreed. What's an appropriate place for this kind of thing? /usr/lib/fwupd? /usr/lib64/fwupd-plugins?

Reply:
It needs to be a path which has the architecture in it. I think the old place plugins were stored seems reasonable.